### PR TITLE
[4.0] mod_languages php notice on error page

### PR DESCRIPTION
**Pull request by brianteeman**
If you have the language switcher module published and you go to a 404 error page then you will see this
image

Apply the patch and the php warning is gone

### CODE REVIEW

File: modules/mod_languages/Helper/LanguagesHelper.php [hunk 1]
```diff
@@ -76,6 +76,11 @@ public static function getList(&$params)
 			else
 			{
 				// Load component associations
+				if (!defined('JPATH_COMPONENT_SITE'))
+				{
+					define('JPATH_COMPONENT_SITE', JPATH_SITE . '/components/' . $app->input->get('option'));
+				}
```
Comment (Quy) [inline modules/mod_languages/Helper/LanguagesHelper.php]:
```suggestion
				}
```

**Comment by SharkyKZ** [comment]
Instead of defining the (un)constant, just replace it with the value. It's deprecated anyways, see #21678.

**Comment by brianteeman** [comment]
lol - it was me that did the deprecation

I will update the pr soon

**Comment by infograf768** [comment]
Folks, this has already been corrected in 3.9.x. It is just a matter to wait for @wilsonge to merge.
See https://github.com/joomla/joomla-cms/pull/22403

**Comment by brianteeman** [comment]
ok thanks - closed

**Pull request closed without merging**